lib/vanagon/project.rb

```diff
@@ -756,7 +756,8 @@ def load_upstream_settings(upstream_project_name, upstream_git_url, upstream_git
         # We don't want to load any of the upstream components, so we're going to
         # pass an array with an empty string as the component list for load_project
         no_components = ['']
-        upstream_project = Vanagon::Project.load_project(upstream_project_name, File.join(working_directory, upstream_source.dirname, "configs", "projects"), platform, no_components)
+        upstream_platform = Vanagon::Platform.load_platform(platform.name, File.join(working_directory, upstream_source.dirname, "configs", "projects"))
```

Comment:
Should this reference the "platforms" directory, rather than "projects"?

Reply:
oo good catch